CondTools/SiPixel/test/SiPixelFakeGenErrorDBSourceReader.cc

```diff
@@ -1,23 +0,0 @@
-#include "CondTools/SiPixel/test/SiPixelFakeGenErrorDBSourceReader.h"
```

Comment:
Why are Fake readers removed?

Reply:
The fake reader are not used at all, maybe I could edit the README to point to an older version of the CMSSW where these are included? For the very low chance when somebody wants to use them?

Reply:
I think they would be nice to keep them, they have a very clear use-case as we discussed privately.